### PR TITLE
wet-370 Conduct WCAG pre-assessment for Toggle (GCWeb)

### PR DESCRIPTION
**Pull request by hongbinyu413**

### CODE REVIEW

File: common/toggle/reports/ally-1-en.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
+	"tag": "toggle",
+	"parentdir": "toggle",
+	"parentPage": "toggle",
+	"parentPageURL": "toggle",
+	"altLangPage": "a11y-1-fr.html",
+	"dateModified": "2023-07-06",
+	"layout": "assessment_wrote_en-en",
+	"reportURL": "pre-a11y-1.json"
```
Comment (duboisp) [inline common/toggle/reports/ally-1-en.html:13]:
```suggestion
	"altLangPage": "ally-1-fr.html",
	"dateModified": "2023-07-06",
	"layout": "assessment_wrote_en-en",
	"reportURL": "pre-ally-1.json"
```

File: common/toggle/reports/pre-ally-1.json [hunk 2]
```diff
@@ -0,0 +1,589 @@
...
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of toggle",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
```
Comment (duboisp) [inline common/toggle/reports/pre-ally-1.json:16]: This CSS selector are not accurate.

```suggestion
            "@value": ".wb-toggle"
```

File: common/toggle/reports/ally-1-fr.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
+	"tag": "toggle",
+	"parentdir": "toggle",
+	"parentPage": "Basculer",
+	"parentPageURL": "toggle",
+	"altLangPage": "a11y-1-en.html",
+	"dateModified": "2023-07-06",
+	"layout": "assessment_wrote_en-fr",
+	"reportURL": "a11y-1.json"
```
Comment (duboisp) [inline common/toggle/reports/ally-1-fr.html:13]:
```suggestion
	"altLangPage": "ally-1-en.html",
	"dateModified": "2023-07-06",
	"layout": "assessment_wrote_en-fr",
	"reportURL": "ally-1.json"
```

**Review by duboisp — CHANGES_REQUESTED** [review]
In addition to the request change, Can you add an index.json-ld file in the folder "common/toggle" like how I did for the share this page like this: https://github.com/wet-boew/GCWeb/blob/master/common/share/index.json-ld

Thanks

**Comment by hanseller** [comment]
I made the changes and push it to  HeansiH: PR2202

**Comment by duboisp** [comment]
https://github.com/HeansiH/GCWeb/commits/PR2202/

**Comment by duboisp** [comment]
I pushed @hanseller changes into this PR

**Review by duboisp — APPROVED** [review]
Reviewed and tested locally. All good.

File: common/toggle/reports/ally-1-en.html (hunk 1 above)
Comment (duboisp) [reply common/toggle/reports/ally-1-en.html]: fixed, idem in French

**Pull request merged**